EDU-2367-EN-Create-API-overview (#302)
aziontech/docs: src/content/docs/en/pages/azion-api/overview/azion-api-overview.mdx

---
title: Azion API
description: Azion RESTful API. Learn more about oficial endpoints, methods and API use cases.
meta_tags: azion, rest api, restful api, azion restful api
namespace: documentation_products_azion_api_overview
permalink: /documentation/products/azion-api-overview/
---

> 
> # Getting Started 

Reviewer: ```suggestion
```

This documentation will guide you on the first steps of using the **Azion API**.

## API overview

> The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com) web platform.

Reviewer: ```suggestion
The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).
```

> > Access the [introduction](https://api.azion.com/#intro) documentation to find reference information about the API.

Reviewer: component tip

Author: ```suggestion
:::tip
> Access the [introduction](https://api.azion.com/#intro) documentation to find reference information about the API.
:::
```

## Prerequisites

To run **Azion API** requests you need:

- **Real-Time Manager (RTM)** account.
- **API token (personal token)**.

> > Learn [how to create an RTM account](https://www.azion.com/en/documentation/products/accounts/creating-account/) and how to create a [personal token](https://www.azion.com/en/documentation/products/accounts/personal-tokens/) to use with the API.

Reviewer: ```suggestion
> Learn [how to create an RTM account](/en/documentation/products/accounts/creating-account/) and how to create a [personal token](/en/documentation/products/accounts/personal-tokens/) to use with the API.
```

Reviewer: componente

Author: ```suggestion
:::tip
> Learn [how to create an RTM account](https://www.azion.com/en/documentation/products/accounts/creating-account/) and how to create a [personal token](https://www.azion.com/en/documentation/products/accounts/personal-tokens/) to use with the API.
:::
```

---
title: Get started with Azion API
description: Learn more about oficial endpoints, methods, and API use cases of the Azion RESTful API.
meta_tags: azion, rest api, restful api, azion restful api
namespace: documentation_products_azion_api_overview
permalink: /documentation/products/azion-api-overview/
---

This documentation will guide you on the first steps of using the **Azion API**.

## API overview

> The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on the [Real-Time Manager (RTM)](https://manager.azion.com).

Reviewer 2: ```suggestion
The **Azion API** is a *RESTful API* based on *HTTPS* requests, that allows users to fully interact with Azion products through API requests. Most API endpoints have the main HTTP methods (`GET`, `POST`, `PUT`, `PATCH`, and `DELETE`), so you can create, read, update, and delete Azion configuration instances. All changes made through the API are reflected on [Real-Time Manager (RTM)](https://manager.azion.com).
```

:::tip

> > Access the [introduction](https://api.azion.com/#intro) documentation to find reference information about the API.

Reviewer 2: ```suggestion
Access the [API introduction](https://api.azion.com/#intro) documentation to find reference information about the API.
```

Reviewer 2: só p tentar ficar mais claro que vai sair dessa doc

:::

## Prerequisites

To run **Azion API** requests you need:

> - **Real-Time Manager (RTM)** account.
> - **API token (personal token)**.

Reviewer 2: ```suggestion
- A **Real-Time Manager (RTM)** account.
- An **API token (personal token)**.
```

:::tip

> > Learn [how to create an RTM account](https://www.azion.com/en/documentation/products/accounts/creating-account/) and how to create a [personal token](https://www.azion.com/en/documentation/products/accounts/personal-tokens/) to use with the API.

Reviewer 2: ```suggestion
See [how to create an RTM account](/en/documentation/products/accounts/creating-account/) and how to create a [personal token](/en/documentation/products/accounts/personal-tokens/) to use with the API.
```

:::

## Make API requests with curl (and other programming languages) snippets
 

> Azion API documentation offers code examples to make requests with the terminal, using the curl tool, or to automate them with programming languages. 

Reviewer 2: ```suggestion
Azion API documentation offers code examples to make requests with the terminal, using the `curl` command, or to automate them with several programming languages.
```

Reviewer 2: a doc oferece ou apenas o Postman, como diz no parágrafo abaixo?

Postman offers HTTPS request snippets for many programming languages.

To make Azion API requests with curl:

1. Access the [Azion API documentation](https://api.azion.com).
2. On the first column, select the endpoint and method you would like to perform. For example `Tools` > `Personal Tokens` > `POST Create Personal Token`.

> 3. Navigate to `Example Request`.

Reviewer 2: ```suggestion
3. Navigate to **Example Request.**
```
